_posts/2020-11-23-demo-day-2-tax-cruncher.md

```diff
@@ -4,22 +4,24 @@ layout: post
 description: How to calculate a taxpayer's liabilities under current law and under a policy reform.
 categories: [demo-days]
```

Comment:
@MaxGhenis What do you think about having more tags if possible?  I think this should definitely have "individual-income-tax" and "tax-cruncher" tags, at least.  I can see a lot of use by filtering out posts that talk about modeling different types of policies or using different models.

Reply:
👍 done